File: src/pattern/bridge/java/Display.java
package pattern.bridge.java;

public class Display {
    DisplayImpl impl;

    public Display(DisplayImpl impl){
        this.impl = impl;
    }

    public void open(){
        this.impl.rawOpen();
    }

    public void print(){
        this.impl.rawPrint();
    }

    public void close(){
        this.impl.rawClose();
    }

    public void display(){
        this.open();
        this.print();
        this.close();
    }
}

File: src/pattern/bridge/kotlin/Display.kt
package pattern.bridge.kotlin

open class Display(var impl: DisplayImpl) {
    fun open(){
        impl.rawOpen()
    }

    fun print(){
        impl.rawPrint()
    }

    fun close(){
        impl.rawClose()
    }

    fun display(){
        this.open()
        this.print()
        this.close()
    }

}